docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Toolbar
+
+### Tutorial
+
+```lua
+-- mod-version:3 -- lite-xl 2.1
+```
+
+This is required to specify Lite XL version compatibility.
```

Comment:
Once again, I think this doesn't have to be explained because it might be explained in a previous tutorial.